modules/mcagar/src/main/ts/ephox/mcagar/api/bdd/TinyUiActions.ts

```diff
@@ -110,5 +110,7 @@ export {
   pWaitForDialog,
   pWaitForPopup,
   pWaitForUi,
-  pTriggerContextMenu
+  pTriggerContextMenu,
+
```

Comment:
why this empty line?

Reply:
Mainly to separate different exports into groups.